pages/linux/settings.md

```diff
@@ -0,0 +1,19 @@
+# settings
+
+> Get information about the Android OS.
+
+- Display a list of settings in the {{global}} namespace:
```

Comment:
Until we have a specific rule of using `{{ }}` in the example descriptions, I'd go with backticks.
```suggestion
- Display a list of settings in the `global` namespace:
```